src/components/Modals/FiatRamps/fiatRampProviders/onramper.ts

```diff
@@ -0,0 +1,120 @@
+import axios from 'axios'
+import { getConfig } from 'config'
+import * as _ from 'lodash'
```

Comment:
can we please use namespaced lodash imports, e.g. `import groupBy from 'lodash/groupBy'` so it tree shakes better